docker-services.yml

```diff
@@ -63,7 +63,7 @@ services:
     environment:
       # settings only for development. DO NOT use in production!
       - bootstrap.memory_lock=true
-      - "OPENSEARCH_JAVA_OPTS=-Xms512m -Xmx512m"
+      - "OPENSEARCH_JAVA_OPTS=-Xms1024m -Xmx1024m"
```

Comment:
I think this is needed only for full migration

Reply:
does it hurt to increase the memory? I like to test the full migration from time to time